guppy-summaries/src/unit_tests/basic_tests.rs

```diff
@@ -10,73 +10,8 @@ use semver::Version;
 use std::collections::BTreeSet;
 
 type Summary = SummaryWithMetadata;
-#[test]
```

Comment:
the diff looks weird, but I just moved the example inputs outside of the testing functions.